HowTos/transitvpc_faq.rst

```diff
@@ -51,115 +51,115 @@ How do I configure an Aviatrix Global Transit Network solution?
 
-How do I delete a spoke GW?
+How do I delete a Spoke GW?
 -----------------------------
 
 Go to the Gateway page, select the gateway you wish to delete and click "Delete." 
 
-An instance in a spoke VPC cannot communicate with the on-prem network. How do I troubleshoot?
+An instance in a Spoke VPC cannot communicate with the on-prem network. How do I troubleshoot?
 -------------------------------------------------------------------------------------------
 
-There are many reasons why an instance in a spoke VPC cannot communicate with the on-prem host or VM. The following troubleshooting steps may be helpful. 
+There are many reasons why an instance in a Spoke VPC cannot communicate with the on-prem host or VM. The following troubleshooting steps may be helpful. 
 
 1. Make sure the `connection between VGW and Transit GW <http://docs.aviatrix.com/HowTos/transitvpc_faq.html#how-do-i-know-if-the-tunnel-between-vgw-and-transit-gw-is-up>`_ is up. 
 
-#. Make sure the CIDR of the on-prem problem subnet (where the VM or host is not reachable from a spoke VPC instance) is propagated to Spoke VPC. That is, make sure the spoke VPC where the problem instance is deployed has `connectivity <http://docs.aviatrix.com/HowTos/transitvpc_faq.html#how-do-i-find-out-what-routes-being-propagated-from-on-prem>`_ to the problem subnet in the on-prem network. 
+#. Make sure the CIDR of the on-prem problem subnet (where the VM or host is not reachable from a Spoke VPC instance) is propagated to Spoke VPC. That is, make sure the spoke VPC where the problem instance is deployed has `connectivity <http://docs.aviatrix.com/HowTos/transitvpc_faq.html#how-do-i-find-out-what-routes-being-propagated-from-on-prem>`_ to the problem subnet in the on-prem network. 
 
 #. Run a traceroute by using an Aviatrix gateway as a test EC2. Launch a t2.micro instance of Aviatrix Gateway from the `Gateway <http://docs.aviatrix.com/HowTos/gateway.html#gateway>`_ at the navigation bar (this gateway is going to be used as a test EC2 instance). Once this gateway is launched, you can run a `traceroute <http://docs.aviatrix.com/HowTos/troubleshooting.html#network-traceroute>`_ from this gateway (test EC2 instance) to the on-prem problem VM. When the test is complete, remember to delete the gateway to conserve consumption. 
 
 #. Do a traceroute from the on-prem problem VM or host to the Aviatrix gateway test EC2 launched in the steps above. 
 
-#. You can do a packet capture by going to Troubleshoot -> Diagnostics -> PACKET CAPTURE. Select the right tunnel interface and run packet capture.  
+#. You can do a packet capture by going to Troubleshoot -> Diagnostics -> Packet Capture. Select the right tunnel interface and run packet capture.  
```

Comment:
PACKET CAPTURE is how it is on the product UI.

Reply:
Got it! How do I discard that change and revert back to "PACKET CAPTURE?"? 

Then, I need to know what the next steps are to update and publish the FAQ. Shall I merge this branch (“Hobbs-Edits-FAQ-Rev2”) with my original edits from "Hobbs-Edits-FAQ?" Will you then merge the "Hobbs-Edits-FAQ" branch with the master branch?

Thanks,
Brian
Mobile: (303) 565-7912

Reply:
I will close the merge request. You can pull a new branch as none of it got merged.